### ISSUE TITLE
subgraph: gasUsed for winningTicketRedeemds is the gas limit on the tx and not the actual gas used

### ISSUE THREAD

**Issue by yondonfu** [livepeer/subgraph#27]
**Describe the bug (required)**

I entered the following query at https://thegraph.com/explorer/subgraph/livepeer/livepeer:

```
{
  winningTicketRedeemeds(first :5, orderBy: blockNumber, orderDirection: desc) {
    gasUsed
  }
}
```

and I noticed that for tx 0xd05bf9676006101c15249494d5a1c83a16d3e8751ae589291d64cdd370b36314 the gasUsed field is 261183, but the gas used shown on the [Etherscan page](https://etherscan.io/tx/0xd05bf9676006101c15249494d5a1c83a16d3e8751ae589291d64cdd370b36314) is 257657. 261183 is actually the gas limit of the tx and not the gas used by the tx.

**Expected behavior (required)**

The gasUsed field should represent the gas used by a tx and not the gas limit of the tx.

**Comment by adamsoffer** [comment]
@0xcadams - According to [this update by the graph team](https://github.com/graphprotocol/graph-node/issues/2619#issuecomment-1116153594), updating graph-cli & graph-ts to their latest versions fixes this issue. Can you confirm?

**Comment by 0xcadams** [comment]
Yeah so the update changed that field name to be gasLimit instead of gasUsed. And then there's the option to have a receipt field to get the gasUsed. We'd need to add a bit more code to get the gasUsed field to be actual gas used and not gas limit

**Comment by rickstaa** [comment]
@mehrdadmms this was implemented but not reviewed in https://github.com/livepeer/subgraph/pull/164. However it will increase subgraph side so the team that takes this on has to look whether adding this is worth the tradeoff.
